securedrop/journalist_app/utils.py

```diff
@@ -134,21 +142,26 @@ def validate_hotp_secret(user: Journalist, otp_secret: str) -> bool:
     :param otp_secret: the new HOTP secret
     :return: True if it validates, False if it does not
     """
+    strip_whitespace = otp_secret.replace(' ', '')
+    secret_length = len(strip_whitespace)
+
+    if secret_length != HOTP_SECRET_LENGTH:
+        flash(ngettext(
+                'HOTP secrets are 40 characters long - you have entered {num}.',
+                'HOTP secrets are 40 characters long - you have entered {num}.',
```

Comment:
As per above, don't want to update strings without a solid reason. Also users are probably going to be cutting/pasting this out of the yubikey management app in tails anyway, so telling them to enter it manually in response to what was probably a mousing snafu seems a little punitive.

Reply:
I should clarify that I'm not wedded to that particular instruction, which could easily be instead (e.g.):

> HOTP secrets are 40 characters long - you have entered {num}. Make sure you type or copy-paste the entire secret.

I think #6197 offers a good general principle (say what just happened, then say what to do next), but I hear you that this may not be the time to introduce this particular string change if it can be avoided.